src/components/multi-select/multi-select.hbs

```diff
@@ -21,17 +21,18 @@
       </svg>
     </div>
   </div>
-  <div class="{{@root.prefix}}--list-box__menu">
+  <fieldset class="{{@root.prefix}}--list-box__menu">
+    <legend class="{{@root.prefix}}--visually-hidden">Description of form elements within the fieldset</legend>
     {{#each items}}
-    <div class="{{@root.prefix}}--list-box__menu-item" id="{{id}}">
+    <div class="{{@root.prefix}}--list-box__menu-item">
       <div class="{{@root.prefix}}--form-item {{@root.prefix}}--checkbox-wrapper">
-        <label for="{{id}}" title="{{label}}" class="{{@root.prefix}}--checkbox-label">
-          <input type="checkbox" name="Option 1" readonly="" tabindex="-1" class="{{@root.prefix}}--checkbox" id="{{id}}" value="on">
+        <label title="{{label}}" class="{{@root.prefix}}--checkbox-label">
```

Comment:
Given that we remove `for` here, how are we linking the `<label>` to the `<input>`?

Reply:
The label is implied by wrapping the input. Which should be totally cool with screen reader tech. I'll make a note to specifically make sure this works as intended in AVT2 testing.